docs/examples/patterns/modal.html

```diff
@@ -34,4 +35,16 @@ <h2 class="p-modal__title" id="modal-title">Help</h2>
       modal.style.display = 'none';
     }
   }
+
+  var showModalButton = document.getElementById('showModal');
+
+  showModalButton.addEventListener('click', function() {
+    showModal('modal');
+  });
+
+  var modal = document.getElementById('modal');
```

Comment:
This is not a good example. Could we target a class on the #showModal element with the correct aria attr to link the toggle and the content please.

Reply:
This is just one of the examples. There is some inconsistency in how elements are targeted in the examples (via id, className, aria) and other issues as well (some are wrapped in self invoked functions, most are not, some target just one instance of the component, others apply to all instances of pattern on the page etc). Probably another round of updates would be worth to make those really consistent.

Will create an issue for that.

Reply:
Fixed here, reported also as #2700